.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: python
+python:
+  - "2.7"
+# Install dependencies dependencies
```

Comment:
Removed this section due to using sudo: false and the new syntax for installing apt packages.